snapcraft/utils.py

```diff
@@ -290,19 +290,27 @@ def prompt(prompt_text: str, *, hide: bool = False) -> str:
 
 
 def humanize_list(
-    items: Iterable[str], conjunction: str, item_format: str = "{!r}"
+    items: Iterable[str],
+    conjunction: str,
+    item_format: str = "{!r}",
+    sort: bool = True,
```

Comment:
I think we should add a quick test for this new parameter

Reply:
Good catch!